natives.json

```diff
@@ -107759,15 +107759,15 @@
 			"build": "1207"
 		},
 		"0xDE4A9B35D028979F": {
-			"name": "_0xDE4A9B35D028979F",
+			"name": "_IS_APP_ACTIVE_BY_NAME",
```

Comment:
This is actually "running", not "active".
Also, I now realize I should've properly named some of these natives, e.g. ``_IS_APP_RUNNING`` should be ``_IS_APP_RUNNING_BY_HASH``.
But I'll fix that later, for now please just name this one ``_IS_APP_RUNNING_BY_NAME``.